run-ab-platform.sh

```diff
@@ -155,7 +160,7 @@ done
 echo
 echo -e "$blue_text""Starting Docker Compose""$default_text"
 
-docker compose up
+docker compose up $dockerDetachedMode
```

Comment:
I think we need to conditionally call `docker compose down` on line 172, right? Currently when I run `./run-ab-platform.sh -b` the containers spin up and then are immediately stopped.

Reply:
@cpdeethree I'm no bash wizard, but I think this should do it: https://github.com/airbytehq/airbyte/pull/26583/commits/19c58f53dbd6ebb9097c5cac73b3e6ec768c2380